src/sass/main.scss

```diff
@@ -60,3 +92,58 @@ $ls-8: -0.45px;
 body {
 	font-family: $ff;
 }
+
+img {
+	max-width: 100%;
+}
+
+// Header
+.heading {
+	@include width-full(375px);
+}
+
+// hero
+.hero-texts {
+	@include width-full(375px);
+}
+
+// hero images
+.hero-imgs {
+	background-color: $clr-neutral-800;
+	position: relative;
+	@include width-full(375px);
+
+	.app-img {
+		position: relative;
+		top: -20%;
```

Comment:
I thought this was going to be absolute as well? Since the .hero-imgs is already relative. Not a correction just an inquiry...

Reply:
this isn't in the last commit. it was a work in progress and some tests. it is removed after. we'll work on that after.

Reply:
Okay no problem at all